src/renderer/frames/cluster-frame/cluster-frame.tsx

```diff
@@ -62,7 +68,16 @@ export const NonInjectedClusterFrame = observer(({
         {
           Component
             ? <Component />
-            : <Redirect to={startUrl.get()} />
+            // NOTE: this check is to prevent an infinite loop
```

Comment:
I don't follow. How could anyone encounter this?

Reply:
People did: https://github.com/lensapp/lens/issues/5462

The bug was that there was a disconnect between the `Redirect` and whether `startUrl` was valid.

Reply:
I did fix the problem as well. This is here to prevent the same problem from occurring again in the future.